### PR TITLE
Refactor out inventory_unit in default populator.

### PR DESCRIPTION
**Pull request by shioyama**
This much doesn't change the API at all.

### CODE REVIEW

**Comment by degicat** [comment]
@miyazawadegica Please review this.

File: app/models/spree/default_license_key_populator.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module Spree
   class DefaultLicenseKeyPopulator < LicenseKeyPopulator
 
     def get_available_keys(inventory_unit, quantity, license_key_type=nil)
-      return false unless count_available(inventory_unit, license_key_type) >= quantity
+      return false unless count_available(license_key_type) >= quantity
       LicenseKey.available.where(
         :variant_id => inventory_unit.variant.id,
```
Comment (miyazawadegica) [inline app/models/spree/default_license_key_populator.rb]: you also change this line `inventory_unit.variant.id` to `variant.id`, right?

**Comment by miyazawadegica** [comment]
Looks good to me :+1:  Please check the above comment.

File: app/models/spree/default_license_key_populator.rb (hunk 1 above)
Comment (shioyama) [inline app/models/spree/default_license_key_populator.rb]: Yes you're right, I'll fix that thanks.

**Pull request merged**